ora/Common/Representables/WindowAccessor.swift

```diff
@@ -35,46 +63,14 @@ struct WindowAccessor: NSViewRepresentable {
     }
 
     func makeNSView(context: Context) -> NSView {
-        let view = NSView()
-
-        DispatchQueue.main.async {
-            guard let window = view.window else { return }
-            isFullscreen = window.styleMask.contains(.fullScreen)
-
-            let coordinator = context.coordinator
-
-            let enterObserver = NotificationCenter.default.addObserver(
-                forName: NSWindow.didEnterFullScreenNotification,
-                object: window,
-                queue: nil,
-                using: coordinator.didEnterFullScreen
-            )
-
-            let exitObserver = NotificationCenter.default.addObserver(
-                forName: NSWindow.didExitFullScreenNotification,
-                object: window,
-                queue: nil,
-                using: coordinator.didExitFullScreen
-            )
-
-            coordinator.observers = [enterObserver, exitObserver]
-            updateTrafficLights(for: window)
-        }
-
-        return view
+        return context.coordinator
```

Comment:
I guess at this point is there much benefit to using the NSView as the coordinator instead of just creating the NSView here?